polkadot/packages/react-query/src/Chain.tsx

```diff
@@ -19,7 +19,7 @@ function Chain ({ children, className = '', label }: Props): React.ReactElement<
 
   return (
     <div className={className}>
-      {label || ''}{systemChain || t('Unknown')}{children}
+      {label || ''}{systemChain?.replace("CENNZnet ", "") || t('Unknown')}{children}
```

Comment:
why do we have to touch the `react-query` to replace the `CENNZnet`?

Reply:
systemName comes from `api.rpc.system.name()` so it's either overriding it in `Api` or editing `react-query`,
same as systemChain which comes from `api.rpc.system.chain()`